lib/licensed/sources/cocoapods.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+require "json"
+require "pathname"
+require "uri"
+require "cocoapods-core"
+
+module Licensed
+  module Sources
+    class Cocoapods < Source
+      def enabled?
+        return unless Licensed::Shell.tool_available?("pod")
+
+        config.pwd.join("Podfile").exist? && config.pwd.join("Podfile.lock").exist?
+      end
+
+      def enumerate_dependencies
+        pods.map { |pod|
+          metadata = dependency_metadata(pod)
+          path = dependency_path(pod.root_name)
+
+          Dependency.new(
+            path: path,
+            name: metadata["name"],
+            version: metadata["version"],
+            metadata: {
+              "type"      => Cocoapods.type,
+              "homepage"  => metadata["homepage"],
+              "summary"   => metadata["summary"],
+              "platforms" => metadata["platform"]
+            }
+          )
+        }
+      end
+
+      private
+
+      def pods
+        if targets.nil?
+          lockfile.dependencies
+        else
+          targets_to_validate = podfile.target_definition_list.filter { |t| targets.include?(t.label) }
+          if targets_to_validate.any?
+            targets_to_validate.map(&:dependencies).flatten
+          else
+            raise Licensed::Sources::Source::Error, "Unable to find any target in the Podfile matching the ones provided in the config."
+          end
+        end
+      end
+
+      def targets
+        config.dig("cocoapods", "targets")&.map { |t| "Pods-#{t}" }
```

Comment:
Yes it will always be prefixed with `Pods-`, it's part of the [nomenclature](https://github.com/CocoaPods/CocoaPods/blob/123f2d02f42075fa9f3b1bc729b342bd844efb44/NOMENCLATURE.md#aggregate-target). Here is the source code responsible for this behaviour ([code](https://github.com/CocoaPods/CocoaPods/blob/98fc505fa0896ba61c0998b32ccae1c577077cc5/lib/cocoapods/installer/analyzer.rb#L382))